lib/form/paragraph3.flow

```diff
@@ -44,6 +44,9 @@ reconstructed on each reflow, because we combine them together to have as few sp
 possible, and because rendering words in the same sprite improves the interword spacing.
 */
 
+INF_METRICS = 100000.0;
+
+NumAlignment ::= Some, Justify;
```

Comment:
Why Some is a part of NumAlignment? looks weird.

Reply:
Only struct allowed as a part of union.